### ISSUE TITLE
False Positive (on using `/` character on identifiers)

### ISSUE THREAD

**Issue by Atreyagaurav** [purcell/package-lint#222]
I have a function named `litex-format-args-/` which uses the character `/` as the divide operator symbol but the separator. But package-lint has it marked as an issue.

Here's the build with error : https://github.com/Atreyagaurav/litex-mode/runs/5127779702?check_suite_focus=true

Here is the definition of function:
```
(defun litex-format-args-/ (args)
  (let ((arg1 (car args))
	(arg-rest (cdr args)))
  (if arg-rest
      (format "\\frac{%s}{%s}"
	      (litex-lisp2latex-all arg1)
              (litex-lisp2latex-all (cons '* arg-rest)))
    (format "\\frac1{%s}" (litex-lisp2latex-all arg1)))))
```

And similar other functions which shows that I'd have to make an exception in the logic for passing this lint test just for divide operator.
```
(defun litex-format-args-1+ (args)
  (concat (litex-lisp2latex-all (car args)) " + 1"))


(defun litex-format-args-expt (args)
  (let ((base (car args))
	(power (cadr args)))
   (if (listp base)
         (format "(%s)^{%s}" (litex-lisp2latex-all base) (litex-lisp2latex-all power))
     (format "%s^{%s}"  (litex-lisp2latex-all base) (litex-lisp2latex-all power)))))


(defun litex-format-args-sqrt (args)
  (format "\\sqrt{%s}" (litex-lisp2latex-all (car args))))
```

Is there a way to make it understand that?

**Comment by purcell** [comment]
No, this isn't configurable. There was a special case for `-/=` at the end of a symbol, and I've added a similar case for `-/`.

**Comment by Atreyagaurav** [comment]
Thank you. It shows `No issues` now.
